selenium_webdriver/pom.xml

```diff
@@ -12,6 +11,99 @@
         <maven.compiler.source>8</maven.compiler.source>
         <maven.compiler.target>8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
 
+        <testng.version>7.4.0</testng.version>
+        <log4j.version>1.2.17</log4j.version>
+        <slf4j-simple.version>1.6.1</slf4j-simple.version>
+        <log4j-api.version>2.11.1</log4j-api.version>
+        <log4j-core.version>2.11.1</log4j-core.version>
+        <maven-surefire-plugin.version>2.22.1</maven-surefire-plugin.version>
+        <selenium-java.version>4.7.2</selenium-java.version>
+        <webdrivermanager.version>5.3.1</webdrivermanager.version>
+        <asertj-core.version>3.24.2</asertj-core.version>
+        <lombok.version>1.18.20</lombok.version>
+        <cucumber-java.version>7.11.1</cucumber-java.version>
```

Comment:
you may use single variable for all cucumber dependencies because they are the same